### PR TITLE
Document how to apply Premailer

### PR DESCRIPTION
**Pull request by vfaronov**
Did I get it right?

### CODE REVIEW

File: docs/transformations.rst [hunk 1]
```diff
@@ -23,6 +23,18 @@ Code example to make images inline:
...
     u'<html><body><img src="cid:promo.png"></body></html>'
 
+To inline CSS using `Premailer <https://github.com/peterbe/premailer>`_:
+
+.. code-block:: python
+
+    >>> html = '''<style>h1 { color: red }</style>
+    ...           <h1>Hello world!</h1>'''
+    >>> message = emails.Message(html=html)
+    >>> message.transformer.premailer.transform()
```
Comment (lavr) [inline docs/transformations.rst]: `Premailer` is a great library, but using it's abstractions here looks a little confusing for me.  

Supposed way to do basic transformations is `Transformer.load_and_transform`:

```
message = emails.Message(html='''<style>h1 { color: red }</style>
         <h1>Hello world!</h1>''')
message.transformer.load_and_transform()
message.transformer.save()
print(message.html)
```

Comment (vfaronov) [inline docs/transformations.rst]: Do you think that this example and the previous one (about images inlining) should be replaced with one bigger example that explains the arguments to `load_and_transform`?

Comment (lavr) [inline docs/transformations.rst]: Not sure. I don't like this code now. I think should add `message.transform` method that incapsulates `load_and_transform` & `save`

Comment (vfaronov) [inline docs/transformations.rst]: Sounds good to me. Well then, I’ll withdraw this PR for now.

Comment (lavr) [inline docs/transformations.rst]: https://github.com/lavr/python-emails/pull/92

**Pull request closed without merging**